### PR TITLE
Products database UI & wire-up

### PR DESCRIPTION
**Pull request by EdiongAlexander**
## Description
- Add products from database

## To Test
Go to add new products and select database entry

## Checklist
- [ ] I have looked over the diffs.
- [ ] There are no errors/failing tests from my changes on this branch.
- [ ] I have requested a review on this PR and re-assigned the Jira ticket.

## Screenshots
image

## Extra Notes
[Please include any extra notes you may have about the changes here.]

### CODE REVIEW

**Review by ObiTam — CHANGES_REQUESTED** [review]
1. In the product database selectore sort by name and use the array index for selection. default to 0.
2. 
image
remove the div within custom scrollbar. - it has no use.
3.
image
your filters are disconnected. here i am trying to filter within proteinous foods for something with "my" instead it filters product database for "my"
You should have a state filters that maps the filtering of product database. that way i can execute multiple filters at a time - by category and name.
4.
I do not think that this is the optimal way to achieve the product database.
no changes need to be made to any file within product-changes, It should all end at addprduct/index.js.
you need a state to switch between manual and database input.
by default product info is initialized with 
```
  const useProductInfo = {
    productCategory: "",
    category: "",
    name: "",
    availability: "0",
    price: 0,
    discount: 0,
    description: "",
    images: [],
    taxable: false,
    unitWeightInGrams: 0,
    pricingType: "per Item",
    measurementUnit: "",
  };
```
add a useEffect that monitors tab and if there is a database input. if a database product has been selected then assign its values to product info. that's all.

File: src/components/auth/forms/login.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ const LogInForm = () => {
       } else {
         const userData = await fetchUserRole("/users/me?populate=*", responseData);
 
-        if (userData?.role?.name === "admin") {
+        if (userData?.role?.name ) {
```
[the remark this answers is not in the document]
Comment (EdiongAlexander) [reply src/components/auth/forms/login.js:56]: I didn't

**Review by ObiTam — COMMENTED** [review]
Hi Kara,

I made some changes to the product modal. 

please see my comments and lets try to get the save as draft and save button to work. tahanks

**Comment by ObiTam** [comment]
closing in favor of other PR

**Pull request closed without merging**